unit_tests/testVTUFiles/testVTUFiles.f90

```diff
@@ -49,92 +50,118 @@ SUBROUTINE SetupError
     ENDSUBROUTINE SetupError
 !
 !-------------------------------------------------------------------------------
-    SUBROUTINE testInitialize
-      !Test Initialize and clear
-      CALL testVTUFile%clear
+    SUBROUTINE testClear()
```

Comment:
In these tests we typically manually set some attributes on the type then call clear.

We then ASSERT everything we can to make sure the object is in its defined uninitialized state. e.g. assert on all attributes (even the inherited ones).